src/Console/Command.php

```diff
@@ -127,6 +125,8 @@ protected function createRunner(): TestRunner
      */
     public function run(array $argv, bool $exit = true): int
     {
+        LoadStructure::in($this->testSuite->rootPath);
+
```

Comment:
This change was necessary in order to ensure that the `Pest.php` file gets loaded before the actual test file itself (see screenshot + explanation below).

<img width="920" alt="Screen Shot 2021-04-07 at 9 36 17 AM" src="https://user-images.githubusercontent.com/18744334/113915779-ce822b00-97a4-11eb-82b2-b59385d2d550.png">

In the first invocation, I am running with a specific directory of tests, which loads `Pest.php` _before_ the test file itself. In the second invocation, I am using the Composer `test:unit` script which does not specify a directory. When this happens, the `parent::run($argv, false)` call on line 130 will load the test file first, causing the test's `uses()` calls to execute before those in the `Pest.php` file.